common-lib/src/test/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessorsTest.scala

```diff
@@ -20,6 +20,14 @@ class SupplierProcessorsTest extends FunSpec with Matchers with MetadataHelper {
     processedImage.metadata.credit should be (Some("Unknown Party"))
   }
 
+  describe("StaffPhotographer") {
+    it("should match StaffPhotographer byline") {
+      val image = createImageFromMetadata("byline" -> "Christopher Thomond")
```

Comment:
I'm a little bit uncomfortable about testing this hard-coded data, but as at the moment it _is_ in the code I feel this is acceptable, (and easier than mocking).